js/gated-content/src/views/VideoPage.vue

```diff
@@ -49,22 +49,26 @@
               <SvgIcon icon="difficulty-icon-white" :css-fill="false"></SvgIcon>
               {{ video.attributes.field_gc_video_level.name | capitalize }}
             </div>
-            <div
-              v-if="video.attributes.field_gc_video_category &&
-              video.attributes.field_gc_video_category.length > 0"
-              class="video-footer__block video-footer__category">
+            <div v-if="videoCategories" class="video-footer__block video-footer__category">
               <SvgIcon icon="categories"
                        class="fill-white"
                        :growByHeight=false></SvgIcon>
-              <span v-for="(category, index) in video.attributes.field_gc_video_category"
-                    :key="category.drupal_internal__tid">
-                <router-link :to="{
-                  name: 'Category',
-                  params: { id: video.relationships.field_gc_video_category.data[index].id }
-                }">
-                  {{ category.name }}
-                </router-link>
-              </span>
+              <ul>
+                <li
+                  v-for="tid in videoCategories"
+                  class="video-footer__category-list-item"
+                  :key="tid"
+                >
+                  <ul>
+                    <li v-for="category in $store.getters.getAncestors(tid)" :key="category.tid">
```

Comment:
We have this code at many places. Maybe is it time to make new component CategoryMenu?